src/all/pandachaika/src/eu/kanade/tachiyomi/extension/all/pandachaika/PandaChaikaDto.kt

```diff
@@ -0,0 +1,104 @@
+package eu.kanade.tachiyomi.extension.all.pandachaika
+
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.model.UpdateStrategy
+import kotlinx.serialization.Serializable
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+@Serializable
+class Archive(
```

Comment:
like previously mentioned, is it not possible to parsing this whole archive in the `/search/` endpoint? because it looks like all the info is already available there

Reply:
Sorry. I can't quite understand. 
Do you mean I should just remove Archive class and use LongArchive, using "/search/xxxx" instead of "?archive=" in mangaDetails and chapterList Request?

Reply:
in popular/latest/search, you call `/search/` which returns list of archives. these archives have all the necessary information (tags/source/category etc etc), so parse everything there (what you do in Archive class). this way (with initialized = true) app won't make additional calls for details.

Reply:
Thank you for the suggestion. Sorry I didn't implement that in the previous commits. 
I thought it wouldn't work because I had already tested it before, but it seems like it's just a cache issue on my end.

Reply:
Hello, I've changed the code as per your suggestion. Please let me know if it's incorrect. I might have misinterpreted your suggestion.